pkg/util/kernel/headers/find_headers.go

```diff
@@ -5,7 +5,8 @@
 
 //go:build linux && linux_bpf
 
-package kernel
+// Package headers is utilities for downloading Linux kernel headers
+package headers
```

Comment:
Do we also want to remove the `header` prefix from the struct names, etc. in this new package?